cmd/zetae2e/local/local.go

```diff
@@ -309,7 +309,8 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 
 		if !skipPrecompiles {
 			precompiledContractTests = []string{
-				e2etests.TestZetaPrecompilesPrototypeName,
+				e2etests.TestPrecompilesPrototypeName,
+				e2etests.TestPrecompilesStakingName,
```

Comment:
Do you think it is fine not implementing a test doing through contract for staking? Is the idea just for prototype to validate the concept of calling precompile with a contract?